pace-util/tests/test_zarr_monitor.py

```diff
@@ -5,25 +5,21 @@
     import zarr
 except ModuleNotFoundError:
     zarr = None
+import copy
+import logging
 from datetime import datetime, timedelta
 
 import cftime
 import pytest
 
-
-try:
-    import xarray as xr
-except ModuleNotFoundError:
-    xr = None
-import copy
-import logging
-
 import pace.util
 from pace.util import X_DIMS, Y_DIMS
+from pace.util._optional_imports import xarray as xr
 from pace.util.testing import DummyComm
 
 
 requires_zarr = pytest.mark.skipif(zarr is None, reason="zarr is not installed")
+requires_xarray = pytest.mark.skipif(xr is None, reason="xarray is not installed")
```

Comment:
Noticed this minor bug where tests would fail instead of skip if xarray is not installed but zarr is installed.